crates/templates/src/writer.rs

```diff
@@ -9,6 +9,7 @@ pub(crate) struct TemplateOutputs {
 pub(crate) enum TemplateOutput {
     WriteFile(PathBuf, Vec<u8>),
     AppendToml(PathBuf, String),
```

Comment:
Would it be worth documenting in line  here what each of the `MergeToml` parts are?